kotest-assertions-arrow-fx-coroutines/src/commonMain/kotlin/io/kotest/assertions/arrow/fx/coroutines/shouldBe.kt

```diff
@@ -1,9 +1,8 @@
 package io.kotest.assertions.arrow.fx.coroutines
 
 import io.kotest.matchers.shouldBe as coreShouldBe
-import kotlin.jvm.JvmName
 
-internal infix fun <A> A.shouldBe(a: A): A {
-  this coreShouldBe a
-  return this
-}
+internal infix fun <A> A.shouldBe(a: A): A =
+  also {
+    this coreShouldBe a
+  }
```

Comment:
https://github.com/kotest/kotest/pull/3186 should make this redundant.. Might be nice to try to get that merged soon so you can remove this :)